client/src/App.test.js

```
@@ -1,5 +1,4 @@
 import React from 'react';
-import { shallow } from 'enzyme';
```

Removing this should throw a 'shallow not defined error'
Is that what you experienced?